src/routes/project/project-route.ts

```diff
@@ -1,11 +1,26 @@
 import { RouteConfig } from 'vue-router';
 
-const Project = () => import(/* webpackChunkName: "ProjectPage" */ '@/views/project/project/pages/ProjectPage.vue');
-const ProjectDetail = () => import(/* webpackChunkName: "ProjectDetail" */ '@/views/project/project/pages/ProjectDetail.vue');
+const ProjectPage = () => import(/* webpackChunkName: "ProjectPage" */ '@/views/project/project/pages/ProjectPage.vue');
+const ProjectDetailPage = () => import(/* webpackChunkName: "ProjectDetailPage" */ '@/views/project/project/pages/ProjectDetailPage.vue');
+
+const ProjectDashboardPage = () => import(/* webpackChunkName: "ProjectDashboardPage" */ '@/views/project/project/pages/ProjectDashboardPage.vue');
+const ProjectMemberPage = () => import(/* webpackChunkName: "ProjectMemberPage" */ '@/views/project/project/pages/ProjectMemberPage.vue');
+const ProjectAlertPage = () => import(/* webpackChunkName: "ProjectAlertPage" */ '@/views/project/project/pages/ProjectAlertPage.vue');
+const ProjectNotificationsPage = () => import(/* webpackChunkName: "ProjectNotificationsPage" */ '@/views/project/project/pages/ProjectNotificationsPage.vue');
+const ProjectMaintenanceWindowPage = () => import(/* webpackChunkName: "ProjectMaintenanceWindowPage" */ '@/views/project/project/pages/ProjectMaintenanceWindowPage.vue');
+const ProjectTagPage = () => import(/* webpackChunkName: "ProjectTagPage" */ '@/views/project/project/pages/ProjectTagPage.vue');
 
 export const PROJECT_ROUTE = Object.freeze({
     MAIN: 'projectMain',
     DETAIL: 'projectDetail',
+    TAB: {
+        SUMMARY: 'projectSummary',
+        MEMBER: 'projectMember',
+        ALERT: 'projectAlert',
+        NOTIFICATIONS: 'projectNotifications',
+        MAINTENANCE_WINDOW: 'projectMaintenanceWindow',
+        TAG: 'projectTag',
+    },
```

Comment:
`TAB`은  `DETAIL` 밑에 있는 것이 어떨까요? 
```suggestion
    DETAIL: {
    MAIN: 'projectDetail',
    TAB: {
        SUMMARY: 'projectSummary',
        MEMBER: 'projectMember',
        ALERT: 'projectAlert',
        NOTIFICATIONS: 'projectNotifications',
        MAINTENANCE_WINDOW: 'projectMaintenanceWindow',
        TAG: 'projectTag',
    }
}
```
얘를 바꾸면 `PROJECT_ROUTE. DETAIL` 로 쓰던 것들도 변경이 같이 일어나야 해서 애매할까요..?

Reply:
좋아요!

Reply:
감사합니다! 다른 라우트 컨피그가 다 이러케 써있어서 맞추려구요 ㅎㅎ